x-pack/plugins/fleet/common/types/models/agent.ts

```diff
@@ -35,7 +35,8 @@ export type AgentActionType =
   | 'UPGRADE'
   | 'SETTINGS'
   | 'POLICY_REASSIGN'
-  | 'CANCEL';
+  | 'CANCEL'
+  | 'UPDATE_TAGS';
```

Comment:
I am kind of confuse by this one do we create a `.fleet-actions` document for `UPDATE_TAGS` ?

Reply:
currently not, I am contemplating to create a document either there or `.fleet-action-status`, so that we could show the status of update tags action as well.
for now added here to use the action type in the action runner